index.html

```diff
@@ -0,0 +1,63 @@
+  <title>Portfolio Skelenton</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+  <link rel='stylesheet' type='text/css' media='screen' href='/css/styles.css'>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link
+	href="https://fonts.googleapis.com/css2?family=Crete+Round:wght@400;500;700&family=Poppins&familty=Inter:wght@500;700&family=Roboto:wght@400;500;700&display=swap&"
+	rel="stylesheet">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.7/css/all.css">
+  <script src="https://code.iconify.design/2/2.1.0/iconify.min.js"></script>
+  <script src="https://kit.fontawesome.com/8f83bc8378.js" crossorigin="anonymous"></script>
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-H9P8Y9RSJ4"></script>
+    <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', 'G-H9P8Y9RSJ4');
+  </script>
+</head>
+<body>
+  <header>
+     <nav class="top-navbar">
+      <a href="#" class="logo">Kemar Norton</a>
+        <ul class="menu-list ul-list">
+          <li class="list-item-text"><a href="#" class="link link-text">Portfolio</a></li>
+          <li><hr class="hr-bar1"></li>
+          <li class="list-item-text"><a href="#" class="link link-text">About</a></li>
+          <li><hr class="hr-bar1"></li>
+          <li class="list-item-text"><a href="#" class="link link-text">Contact</a></li>
+          <li><hr class="hr-bar1"></li>
+          <li class="list-item-text"><a href="#" class="link link-text1 fas fa-envelope"></a></li>
+        </ul>
+        <div class="menuIcon">
+          <span class="bar-1"></span>
+          <span class="bar-2"></span>
+          <span class="bar-3"></span>
+        </div>
+    </nav>
+  </header>
+  <section class="first-page">
+    <h1 class="first-pagetitle">Hey There. <br> I'm Kemar</h1>
+    <h3 class="subtitle">I'm a Software Developer</h3>
+    <p class="description-text">I can help you build a product, feature or website Look through some of my work and
+      experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <ul class="social-mediaIcons">
+      <li class="list-item-icon"><a href="#" class="link-icon fa fa-github"></a></li>
+      <li class="list-item-icon"><a href="#" class="link-icon fa fa-linkedin"></a></li>
+      <li class="list-item-icon"><a href="#" class="link-icon fa fa-angellist"></a></li>
+      <li class="list-item-icon"><a href="#" class="link-icon fa fa-twitter"></a></li>
+      <li class="list-item-icon"><a href="#" class="link-icon fab fa-medium-m"></a></li>
+    </ul>
+  </section>
+    <script src="/js/main.js"></script>
```

Comment:
- [ ] For some reason your js file is not been loaded. Try correcting your file path and verify that your mobile menu functions as expected before asking for another review.